src/live/util.rs

```diff
@@ -62,8 +61,7 @@ pub(super) fn write_live_iso(
                 .tempfile_in(output_dir)
                 .context("creating temporary file")?;
             input.seek(SeekFrom::Start(0)).context("seeking input")?;
-            input
-                .copy_to(output.as_file_mut())
+            std::io::copy(input, output.as_file_mut())
```

Comment:
We're already importing `std::io::copy`.

Reply:
Ah, copy that!